README.md

```diff
@@ -2,7 +2,9 @@
 
 [![Contribute](https://che.openshift.io/factory/resources/factory-contribute.svg)](https://che.openshift.io/f?url=https://github.com/redhat-developer/devfile)
 
-This repository contains the specs, docs and examples of the devfile v1. Devfile v2 specification doc is [here](https://devfile.github.io/website/devfile/index.html) instead. 
+Devfile v1 specs docs and examples are located here.
+
+For Devfile v2, spec, docs and examples are located [here](https://devfile.github.io/devfile/index.html) instead. 
```

Comment:
Maybe is just me, but that sense looks really weird

Reply:
it's not only you